social-messenger-ts/src/components/MessagingChannelList/MessagingChannelList.tsx

```diff
@@ -1,22 +1,18 @@
-import React, { useEffect } from 'react';
+import React, { FC, PropsWithChildren, useEffect } from 'react';
 import { ChannelListMessengerProps, useChatContext } from 'stream-chat-react';
 
 import './MessagingChannelList.css';
 import { SkeletonLoader } from './SkeletonLoader';
 
 import type { StreamChat } from 'stream-chat';
-
 import type { StreamChatGenerics } from '../../types';
 
-const MessagingChannelList: React.FC<ChannelListMessengerProps> = (props) => {
+const MessagingChannelList: FC<PropsWithChildren<ChannelListMessengerProps>> = (props) => {
```

Comment:
Let's get rid of the FC completely and just directly type the props - I remember we had a bunch of issues with it when updating our SDK to R18.

Reply:
Fixed, please see 0651e25